README.md

```diff
@@ -1,6 +1,7 @@
 # jv-salary-info
 
-Implement method `getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)`. It should calculate the salary for employees. As input, you receive two arrays and two dates in String format. 
+Implement method `getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)`. 
+It should calculate the salary for employees. As input, you receive two arrays and two dates in String format. 
```

Comment:
undo unintended changes, be careful

Reply:
not fixed, copy the exact text from master, you can use IDE's feature to compare your current file with the one from some branch